drivers/kscan/Kconfig.ft5336

```
@@ -24,4 +24,10 @@ config KSCAN_FT5336_INTERRUPT
 	help
 	  Enable interrupt support (requires GPIO).
 
+config KSCAN_FT5336_POWER_GPIO
+	bool "Enable Power pin"
```

As per the guidelines, please remove <word>:
https://docs.zephyrproject.org/latest/guides/build/kconfig/tips.html#prompt-strings

> For a Kconfig symbol that enables a driver/subsystem FOO, consider having just “Foo” as the prompt, instead of “Enable Foo support” or the like.

Maybe just write:
```suggestion
	bool "Power pin wake-up"
```